experimental/trusted_shuffler/backend/src/main.rs

```diff
@@ -53,9 +71,14 @@ async fn handler(request: Request<Body>) -> Result<Response<Body>, hyper::Error>
     }
 }
 
-#[tokio::main]
+#[tokio::main(flavor = "multi_thread", worker_threads = 2)]
```

Comment:
Do you know if it's possible to specify the number of threads in the binary agrument?

Reply:
Why only 2 threads anyways? It seems that the default config should generally be more optimal anyways: https://docs.rs/tokio/latest/tokio/attr.main.html#multi-threaded-runtime